_includes/_acquire.htm

```diff
@@ -60,44 +66,20 @@ <h4 class="text-bold">
                 <ul>
                     <li>
-                        <ul>
-                            <li>
-                                In Gridcoin's Proof-of-Stake (POS) system, rewards are minted at a rate of 10 Gridcoin per block. Users earn their BOINC rewards when they stake a POS block.
-                            </li>
-                            <li>
-                                In order to stake a block (and earn your BOINC rewards), a positive balance of GRC is required (This is the case for all POS cryptocurrencies).
-                                <ul>
-                                    <li>
-                                        There are 960 blocks per day that you can attempt to stake. A user with 1% of the total Gridcoin supply will stake approximately 10 blocks per day if 100% of coins are actively staking. Given that we don't experience 100% staking participation, your staking frequency estimation will likely be an underestimation.
-                                    </li>
-                                    <li>
-                                        The formula required to work out the quantity of coins to stake in a period of time is:
-                                        <ul>
-                                            <li>
-                                                <code>
-                                                EstimatedDaysUntilStake=(1/(UserBalance/NetworkCoinSupply))/960
-                                                </code>
-                                            </li>
-                                            <li>
-                                                <code>
-                                                CoinsRequired=(1/(days*960))*NetworkCoinSupply
-                                                </code>
-                                            </li>
-                                        </ul>
-                                    </li>
-                                    <li>
-                                        The payout window is approximately 182.5 days. A staking frequency greater than this is potentially insufficient for solo-staking.
-                                    </li>
-                                    <li>
-                                        Thusly, the rate at which you are rewarded for your BOINC work is identical to the rate at which you mint POS blocks (since you're rewarded for DPOR on top of your POS payment).
-                                    </li>
-                                    <li>
-                                        Note: Your balance does not affect the magnitude calculation, only the rate of payment.
-                                    </li>
-                                </ul>
-                            </li>
-                        </ul>
+                        In Gridcoin's Proof-of-Stake (POS) system, BOINC rewards are given when users stake, along with an additional 10 GRC.
```

Comment:
(POS) should be (PoS) as that's usually how the acronym goes. Maybe it's just my preference though.

Reply:
Didn't catch that, will fix all the occurrences

Reply:
POS is also acceptable